integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/send/SendIntegrationTestCase.java

```diff
@@ -630,9 +630,10 @@ public void testSendingFailOverEndpoint_Receiving_Sequence_LocalReg_BuildMessage
         }
     }
 
+    // Temporarily disabling the test case due to random build failure in the jenkins
     @SetEnvironment(executionEnvironments = {ExecutionEnvironment.STANDALONE
 })
-    @Test(groups = "wso2.esb", description = "Test sending request to Fail Over Endpoint Receiving Sequence in Config Registry while BuildMessage enabled")
+    //@Test(groups = "wso2.esb", description = "Test sending request to Fail Over Endpoint Receiving Sequence in Config Registry while BuildMessage enabled")
```

Comment:
We can add @Test(enabled = false) instead of commenting

Reply:
This is just a temporary action until we find the root cause for this.